source/solvers/vof.cc

```diff
@@ -663,79 +665,55 @@ VolumeOfFluid<dim>::find_sharpening_threshold()
       if (this->simulation_parameters.multiphysics.vof_parameters.conservation
             .verbosity != Parameters::Verbosity::quiet)
         {
-          this->pcout << "   ... step " << nb_search_ite
-                      << " of the search algorithm" << std::endl;
+          this->pcout
+            << "   ... step " << nb_search_ite
+            << " of the search algorithm, min, avg, max mass deviation is : "
+            << mass_deviation_min << " " << mass_deviation_avg << " "
+            << mass_deviation_max << std::endl;
```

Comment:
I disagree here. I actually leave it to always on, it's very useful to monitor how the convergence is progressing and can really help to set the max and min.

Reply:
Ok, can you then update the documentation? It would be useful then to explain what these values are for the users, and how to use them to set the max and min (which are still centered around `0.5`, right?).

Also, if you want to make `verbosity = verbose` the default here, do not forget to update the code and documentation accordingly :)

Reply:
I'm not sure what you mean by update the documentation?

Reply:
On the VOF page of the documentation I put an example of what `verbose` would output. You just need to update that to reflect your changes :)